make.paws/tests/testthat/test_docs.R

```diff
@@ -564,7 +566,11 @@ test_that("convert", {
   expect_equal(convert(text), expected)
 
   text <- "<body>foo \\bar { \\u0123 <code>baz'</code></body>"
-  expected <- "foo \\\\bar \\{ `U+0123` `baz\\'`"
+  # TODO(davidkretch): The following commented out expected value passes on
```

Comment:
Has this TODO been resolved now?

Reply:
No, sadly. I’m not sure what to do here but choose the option where it passes on GitHub actions.

Reply:
Maybe i should just get rid of the comment.